en/config/battery.md

```diff
@@ -0,0 +1,81 @@
+# Battery and Powermodule Setup
+
```

Comment:
Perhaps change introduction to this (the "span" is required by gitbook to properly render notes that follow each other.

---
```
The power setup for PX4 allows you to monitor the battery state and determine remaining available flight time. 

> **Warning** Flying without sufficient power may lead to deep discharge of the battery. This can damage the battery and may lead to loss of vehicle control/crashes.

<span></span>
> **Note** PX4 battery monitoring requires hardware (typically a power module) to measure the battery voltage and the current between battery and vehicle. 
```

Reply:
I didn't know, this time I saved the effort to have a gitbook rendered preview and worked with a simple mark down editor to save time...

Reply:
No worries!